app/views/transactions/index.html.slim

```diff
@@ -1,3 +1,7 @@
+.row
+  .col-md-10
+    h5.nav.navbar-text.mr-1= t('.balance')
+    h5.nav.navbar-text.mr-4= number_to_currency(current_user.user_balance.to_i) if current_user
```

Comment:
@sinpai 
```ruby
h5.nav.navbar-text.mr-1= t('.balance')
h5.nav.navbar-text.mr-4= number_to_currency(current_user.user_balance.to_i) if current_user
```
is duplicated in two placeses, what do you think about DRY partial way?